airflow-core/src/airflow/ui/public/i18n/locales/nl/common.json

```
@@ -7,30 +7,121 @@
     "Providers": "Providers",
     "Variables": "Variabelen"
   },
+  "asset_one": "Asset",
+  "asset_other": "Assets",
   "assetEvent_one": "Asset Event",
   "assetEvent_other": "Asset Events",
+  "backfill_one": "Backfill",
+  "backfill_other": "Backfills",
   "browse": {
     "auditLog": "Audit Log",
+    "requiredActions": "Vereiste acties",
     "xcoms": "XComs"
   },
+  "collapseDetailsPanel": "Details paneel inklappen",
```

"paneel" feels obsolete here

```suggestion
  "collapseDetailsPanel": "Details inklappen",
```